rapid7_insightvm/komand_rapid7_insightvm/util/resource_helpers.py

```diff
@@ -121,41 +130,57 @@ def validate_user_permissions(self, console_url: str, user: dict) -> dict:
         :param user: dict containing the details for a user (from the InsightVM API or matching the format)
         :return: The user dict with updated permissions if required
         """
-        endpoint = endpoints.Role.roles(console_url, user['role']['id'])
+        try:
+            endpoint = endpoints.Role.roles(console_url, user['role']['id'])
+        except KeyError:
+            raise PluginException(cause=self._BADJSON,
+                                  assistance=self._SUPPORT)
```

Comment:
I think @jschipp-r7 's original comment was asking to use the preset, so `raise PluginException(preset=PluginException.Preset.INVALID_JSON`